e2e/smoke/src/dependency/proto/dependency.proto

```
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "smoke.dependency";
+
+package smoke.dependency;
```

Buf (1.72.0)

[error] Files with package "smoke.dependency" must be within a directory "smoke/dependency" relative to root but were in directory "e2e/smoke/src/dependency/proto".

(PACKAGE_DIRECTORY_MATCH)